### DISCUSSION TITLE
Teslamate LXC script

### DISCUSSION THREAD

**Discussion by Jens-Wymeersch** [tteck/Proxmox#1078] [General]
Dear @tteck, 

Your scripts are amazing... Every time I find something new and useful. 
I'm an owner of a Tesla. Although that I have already an integration in HomeAssistant, the Teslamate platform (https://github.com/adriankumpf/teslamate) looks amazing. That said, somehow I never got Grafana to work and it would be great to have it as a LXC.

Probably if you don't have a Tesla this might not be of interest...

Jens

**Comment by tteck** [comment]
https://docs.teslamate.org/docs/installation/debian

**Reply by Jens-Wymeersch** [reply]
@tteck thank you for your quick reply. I assume you aren't interested in making one of your amazing scripts related to this ?

**Comment by SergiCEES** [comment]
I would be interested to! ;)

**Comment by lolobar** [comment]
That would be nice ;-)

**Comment by leah-potato** [comment]
Same here!
